linalg/operator.hpp

```diff
@@ -309,30 +309,37 @@ class Operator
 
 
 /// Base abstract class for first order time dependent operators.
-/** Operator of the form: (x,t) -> f(x,t), where k = f(x,t) generally solves the
-    algebraic equation F(x,k,t) = G(x,t). The functions F and G represent the
-    _implicit_ and _explicit_ parts of the operator, respectively. For explicit
-    operators, F(x,k,t) = k, so f(x,t) = G(x,t). */
+/** Operator of the form: (u,t) -> k(u,t), where k generally solves the
+    algebraic equation F(u,k,t) = G(u,t). The functions F and G represent the
+    _implicit_ and _explicit_ parts of the operator, respectively.
+
+    A common use for this class is representing a differential algebraic
+    equation of the form \f$ F(y,\frac{dy}{dt},t) = G(y,t) \f$. Consider an
+    ordinary differential equation of the form \f$ M \frac{dy}{dt} = g(y,t) \f$,
+    where F and G are defined as one of the following:
+    1. F(u,k,t) = k and G(u,t) = inv(M) g(u,t)
+    2. F(u,k,t) = M k and G(u,t) = g(u,t)
```

Comment:
You are changing `x` to `u` -- why use `y` here and not `u`? It is best to stay consistent.

Reply:
I tried to stay consistent with using $F(u,k,t)$ and $G(u,t)$ for general algebraic equations, reserving $y$ for when the specialized case when the algebraic equations are represent either an ODE or a DAE.

Reply:
My understanding is that `TimeDependentOperator` can be used for things other than DAEs/ODEs... so I reserved the $u$ for full generality and $y$ when `TimeDependentOperator` represents a DAE/ODE.

Reply:
Okay, I see. It's hard to convey that intent in the text without stating it explicitly, at least I didn't get the idea. 😁

However, as I said above, it is a minor thing, so don't feel like you need to change `y` to `u`.

Reply:
I'll also note I found using `u` as the independent variable that `F` depends on and `y` the solution to the DAE/ODE to be useful in explaining `ImplicitSolve`